useIsShake.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react';
+import { Accelerometer } from 'expo-sensors';
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_LOAD_DURATION, SVG_OUT_DURATION, SVG_IN_DURATION, SVG_IN_DELAY, TRANSITION_TIME_2 } from './constants';
+
+export function useIsShake() {
+  const [{ x/*, y, z*/ }, setData] = useState({
+    x: 0,
+    /*y: 0,
+    z: 0,*/
```

Comment:
In general don't commit commented out code to `main`. There are exceptions for this, but this wouldn't qualify